src/ontology/mondo-edit.obo

```diff
@@ -570175,6 +570185,3049 @@ intersection_of: MONDO:0700097 MONDO:0000721 ! cross-species analog xanthinuria
+is_a: MONDO:1011336 {source="OMIA:001723", source="https://orcid.org/0000-0002-5002-8648"} ! nervous system disorder, non-human animal
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7225" xsd:anyURI
+
+[Term]
+id: MONDO:1011890
+name: leukoencephalomyelopathy, non-human animal
+xref: OMIA:001788 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:001788"} ! non-human animal disease
+is_a: MONDO:1011336 {source="OMIA:001788", source="https://orcid.org/0000-0002-5002-8648"} ! nervous system disorder, non-human animal
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7225" xsd:anyURI
+
+[Term]
+id: MONDO:1011891
+name: cervical vertebral compressive myelopathy, non-human animal
+xref: OMIA:001894 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:001894"} ! non-human animal disease
+is_a: MONDO:1011336 {source="OMIA:001894", source="https://orcid.org/0000-0002-5002-8648"} ! nervous system disorder, non-human animal
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7225" xsd:anyURI
+
+[Term]
+id: MONDO:1011892
+name: spinal intradural arachnoid cyst, non-human animal
+xref: OMIA:001923 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:001923"} ! non-human animal disease
+is_a: MONDO:1011336 {source="OMIA:001923", source="https://orcid.org/0000-0002-5002-8648"} ! nervous system disorder, non-human animal
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7225" xsd:anyURI
+
+[Term]
+id: MONDO:1011893
+name: neurodegenerative vacuolar storage disease, non-human animal
+xref: OMIA:001954 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:001954"} ! non-human animal disease
+is_a: MONDO:1011336 {source="OMIA:001954", source="https://orcid.org/0000-0002-5002-8648"} ! nervous system disorder, non-human animal
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7225" xsd:anyURI
+
+[Term]
+id: MONDO:1011894
+name: degenerative encephalopathy, non-human animal
+xref: OMIA:002055 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:002055"} ! non-human animal disease
+is_a: MONDO:1011336 {source="OMIA:002055", source="https://orcid.org/0000-0002-5002-8648"} ! nervous system disorder, non-human animal
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7225" xsd:anyURI
+
+[Term]
+id: MONDO:1011895
+name: Guillain-Barr-like polyradiculoneuropathy, non-human animal
+xref: OMIA:002062 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:002062"} ! non-human animal disease
+is_a: MONDO:1011336 {source="OMIA:002062", source="https://orcid.org/0000-0002-5002-8648"} ! nervous system disorder, non-human animal
+property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7225" xsd:anyURI
+
+[Term]
+id: MONDO:1011896
+name: syndrome des veaux tourneurs, non-human animal
```

Comment:
This is a french name. Do you know if we had an english one? 
the translation would be something like "twirling calf syndrome"

Reply:
English name is Turning calves syndrome. Since it only occurs in cattle, I am deleting this term and will instead only have 'Turning calves syndrome, cattle' term (to be added in a separate PR).